shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -380,11 +380,17 @@ bool FlutterWindowsEngine::Run(std::string_view entrypoint) {
 
   args.custom_task_runners = &custom_task_runners;
 
+  if (!platform_view_manager_) {
```

Comment:
It looks like `platform_view_manager_` is always null. Should there be a constructor argument for `platform_view_manager_`? Or should the platform view manager be created unconditionally? I don't feel strongly about this, feel free to keep as is

Reply:
My intention was that it would be created unconditionally, unless we are in a test case where it is provided beforehand, as is the case in this PR's unit test.

Reply:
The `FlutterWindowsEngineTest.ReceivePlatformViewMessage` test creates a real platform view manager and then later overrides it with the mock. The test should still work as expected if this `if` is removed

Reply:
Can you elaborate on your reasoning? This code runs in `Run`, not the constructor. By the time the `Run` method is called in the unit test, the platform view manager has already been set to the mock, so this if statement is not triggered. Debugging confirms that only one platform view manager is created for the unit test.

Reply:
Oh I'm sorry, I completely missed this was `Run` and not the constructor! Apologies for the confusion, marking as resolved